src/index.html

```diff
@@ -4,9 +4,72 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="./styles/style.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length--28 calendar--start-day--tue">
+      <div class="calendar__block"></div>
+
+      <div class="calendar__block"></div>
```

Comment:
These spaces are not needed, because these tags are not multiline
```suggestion
      <div class="calendar__block"></div>
      <div class="calendar__block"></div>
```